packages/auto-label/src/auto-label.ts

```diff
@@ -112,6 +112,12 @@ export function autoDetectLabel(
   firstPart = firstPart.toLowerCase(); // Convert to lower case.
   firstPart = firstPart.replace(/\s/, ''); // Remove spaces.
 
+  // The Conventional Commits "docs:" prefix is far more common than "api: docs".
+  // So, never label with "api: docs".
+  if (firstPart === 'docs') {
```

Comment:
This happens with `build` too 😆

Reply:
I agree it might be worth adding `build` too.

Reply:
Done! I also updated the logic to short-circuit for titles that start with `docs:` or `build:`.